src/Controls/src/Core/HandlerImpl/Button/Button.Windows.cs

```diff
@@ -5,11 +5,37 @@
 using Microsoft.Maui.Controls.Internals;
 using Microsoft.Maui.Controls.Platform;
 using Microsoft.Maui.Handlers;
+using Microsoft.UI.Xaml;
+using WButton = Microsoft.UI.Xaml.Controls.Button;
 
 namespace Microsoft.Maui.Controls
 {
 	public partial class Button
 	{
+		WButton _button;
+
+		private protected override void OnHandlerChangedCore()
+		{
+			base.OnHandlerChangedCore();
+
+			if (Handler is not null)
+			{
+				if (Handler is ButtonHandler buttonHandler && buttonHandler.PlatformView is WButton button)
+				{
+					_button = button;
+					_button.SizeChanged += OnButtonSizeChanged;
+				}
+			}
+			else
+			{
+				if (_button is not null)
+				{
+					_button.SizeChanged -= OnButtonSizeChanged;
+					_button = null;
+				}
+			}
```

Comment:
Lets reduce nested code as much as possible.
```suggestion
			else if (_button is not null)
			{
				_button.SizeChanged -= OnButtonSizeChanged;
				_button = null;
			}
```